octobot_commons/config_manager.py

```diff
@@ -254,35 +277,35 @@ def clear_dictionaries_by_keys(dict_dest, dict_src):
     return dict_dest
 
 
-# @staticmethod
-# def _update_activation_config(to_update_data, current_config, config_file_path, config_file, deactivate_others):
-#     from tentacles_management.class_inspector import get_class_from_string, evaluator_parent_inspection
-#     something_changed = False
-#     for element_name, activated in to_update_data.items():
-#         if element_name in current_config:
-#             active = activated if isinstance(activated, bool) else activated.lower() == "true"
-#             current_activation = current_config[element_name]
-#             if current_activation != active:
-#                 get_logger().info(f"{config_file} updated: {element_name} "
-#                                   f"{'activated' if active else 'deactivated'}")
-#                 current_config[element_name] = active
-#                 something_changed = True
-#     if deactivate_others:
-#         import evaluator.Strategies as strategies
-#         for element_name, activated in current_config.items():
-#             if element_name not in to_update_data:
-#                 if current_config[element_name]:
-#                     # do not deactivate strategies
-#                     config_class = get_class_from_string(element_name, strategies.StrategiesEvaluator,
-#                                                          strategies, evaluator_parent_inspection)
-#                     if config_class is None:
-#                         get_logger().info(f"{config_file} updated: {element_name} "
-#                                           f"{'deactivated'}")
-#                         current_config[element_name] = False
-#                         something_changed = True
-#     if something_changed:
-#         with open(config_file_path, "w+") as config_file_w:
-#             config_file_w.write(dump_json(current_config))
+def _update_activation_config(to_update_data, current_config, config_file_path, config_file, deactivate_others):
+    from octobot_commons.tentacles_management.class_inspector import get_class_from_string, evaluator_parent_inspection
+    something_changed = False
+    for element_name, activated in to_update_data.items():
+        if element_name in current_config:
+            active = activated if isinstance(activated, bool) else activated.lower() == "true"
+            current_activation = current_config[element_name]
+            if current_activation != active:
+                get_logger().info(f"{config_file} updated: {element_name} "
+                                  f"{'activated' if active else 'deactivated'}")
+                current_config[element_name] = active
+                something_changed = True
+    if deactivate_others:
+        import evaluator.Strategies as strategies
```

Comment:
Where does `evaluator.Strategies` come from ?

Reply:
from my pycharm mixing 2 projects 🤣  and the web interface not working yet on this side (didn't see it)

Reply:
It will be migrated in https://github.com/Drakkar-Software/OctoBot-Evaluators/issues/10 and https://github.com/Drakkar-Software/OctoBot-Trading/issues/35

Reply:
:+1: